nnvm/python/nnvm/testing/tf.py

```diff
@@ -227,3 +227,31 @@ def get_workload_inception_v1():
         graph_def.ParseFromString(f.read())
         graph = tf.import_graph_def(graph_def, name='')
         return (image_data, tvm_data, graph_def)
+
+def get_workload_mobilenet():
```

Comment:
get_workload_inception_v1() get_workload_inception_v3() get_workload_mobilenet()
these functions can be merged into a unified one that takes url and returns a tensorflow model

Reply:
Unified the model import across all these.